crypto/bio/bio_sock.c

```diff
@@ -214,35 +214,7 @@ int BIO_socket_ioctl(int fd, long type, void *arg)
 {
     int i;
 
-#  ifdef __DJGPP__
-    i = ioctlsocket(fd, type, (char *)arg);
-#  else
-#   if defined(OPENSSL_SYS_VMS)
```

Comment:
I won't be surprised if VMS went the way of the dodo soon, but that's unrelated to this removal. So I'd keep this code...

Reply:
True, I'll put it back for now.